source/_cookbook/perform_actions_based_on_input_select.markdown

```diff
@@ -75,3 +75,96 @@ automation:
       data:
         entity_id: media_player.nursery
 ```
+A little bit more complex example that uses [`input_select`](/components/input_select/) and template do decide what and in which  [Chromecast](components/media_player.cast/) play.
+
+```yaml
+input_select:
+  radio_station:
+    name: Radio Station
+    options:
+      - Z88.3
+      - Virgin
+      - RMC
+      - rmcHQ
+      - 105
+      - None
+    initial: None
+    icon: mdi:radio
+  radio_player:
+    name: Radio Player
+    options:
+      - Mansarda
+      - Doccia
+      - Bed
+      - Bath
+      - Salotto
+      - Salotto Video
+      - None
+    initial: None
+    icon: mdi:airplay
+
+automation:
+  - alias: Stop Streaming Radio
+    trigger:
+      - platform: state
+        entity_id: input_select.radio_station
+        to: "None"
+    action:
+      service: media_player.turn_off
+      data_template:
+        entity_id: >
+            {% if is_state("input_select.radio_player", "Mansarda") %}
```

Comment:
You need to escape `{% raw %}{ ... }{% endraw %}` the template parts otherwise they are rendered.